README.md

```diff
@@ -0,0 +1,19 @@
+## YaST - Sudo Configuration
+
+Intention of this module is to provide UI for configuring sudo.
+
+### Known Limitations
+
+- Module using for parsing sudoers own handcrafted perl parser which has
```

Comment:
Something like: We use own hand-crafted parser written in Perl for reading and writing the configuration. It has some limitations especially regarding changes in configuration format introduced in sudo X.Y.Z. Augeas lenses are currently more up-to-date.